lib/main_common.pm

```diff
@@ -255,6 +256,17 @@ sub load_slepos_tests {
     }
 }
 
+sub load_docker_tests {
+    loadtest "console/docker";
+    loadtest "console/runc";
+    # No package 'docker-compose' in SLE
+    if (is_leap || is_tumbleweed) {
```

Comment:
I wouldn't do that. We have also something like "krypton" and "argon". Also I am not so sure about "is_tumbleweed": It is sometimes used as a distribution check, sometimes version check and if it would be for me I would try to avoid it at all because tumbleweed should be our default which never appears anywhere. Instead just specify the distribution in general (as we had in before with `check_var('DISTRI', 'opensuse')` or just the exceptions, e.g. `!is_sle`

Reply:
@mnowaksuse sorry for the bad idea about TW :)

@okurz: so it's better to use `!is_sle` to test *all-but-SLE* and avoid using `is_leap` and `is_tumbleweed`?

Reply:
In this case I think so, yes.

Reply:
+1 to @okurz , @mnowaksuse please change line to !is_sle and solve conflicts and then it can be merged. Sorry for this back and forth it happens sometimes when people looking for better solution :)

Reply:
That's alright, ready to be merged now.